dev/packages/alpha/netflow-0.0.2/kibana/dashboard/34e26884-161a-4448-9556-43b5bf2f62a2.json

```diff
@@ -0,0 +1,294 @@
+{
+    "attributes": {
+        "description": "Overview of Netflow",
+        "hits": 0,
+        "kibanaSavedObjectMeta": {
+            "searchSourceJSON": {
+                "filter": [
+                    {
+                        "$state": {
+                            "store": "globalState"
+                        },
+                        "meta": {
+                            "alias": null,
+                            "disabled": false,
+                            "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.filter[0].meta.index",
+                            "key": "input.type",
+                            "negate": false,
+                            "params": {
+                                "query": "netflow"
+                            },
+                            "type": "phrase",
+                            "value": "netflow"
+                        },
+                        "query": {
+                            "match": {
+                                "input.type": {
```

Comment:
I wonder if this query will still be needed as I assume all the data in the index will be related to netflow. Should this be replaced by a query for type + dataset?